modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -484,9 +484,16 @@ public PartitionReplicaListener(
                 clusterNodeResolver,
                 replicationGroupId,
                 localNode,
-                txRecoveryEngine
-        );
+                txRecoveryEngine);
+
+        buildIndexReplicaRequestHandler = new BuildIndexReplicaRequestHandler(
+                indexMetaStorage,
+                txRwOperationTracker,
+                safeTime,
+                raftCommandApplicator);
```

Comment:
```suggestion
                raftCommandApplicator
                );
```

Reply:
I would prefer to write `);` on the same line as the last parameter is placed. It allows to save one line and does not break anything, including readability.